cmd/generate/codeowners/output.go

```diff
@@ -8,18 +8,30 @@ import (
 	"strings"
 
 	"github.com/open-sauced/pizza-cli/pkg/config"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 )
```

Comment:
>   cmd/generate/codeowners/output.go:22: File is not `goimports`-ed (goimports)

You'll want to run `just format` to get the imports ordered correctly. This is why linting is failing!

Reply:
Maybe we need a `just dev` command that runs all the necessary pre-commit things (like build, test, format, and lint)

Reply:
Looks like the dev instructions are missing the step for installing `goimports`:

```sh
go install golang.org/x/tools/cmd/goimports@latest
```

https://pkg.go.dev/golang.org/x/tools/cmd/goimports

Maybe it also makes sense to have a `just dev-install` command that installs necessary tooling as an easy one off for onboarding.